plugins/browser-plugin-webview/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import { BrowserPlugin, BrowserTracker, Payload } from '@snowplow/browser-tracker-core';
+import { hasMobileInterface, trackWebViewEvent } from '@snowplow/webview-tracker';
+import { Logger, SelfDescribingEvent, SelfDescribingJson } from '@snowplow/tracker-core';
+
+const _trackers: Record<string, BrowserTracker> = {};
+
+export function WebViewPlugin(): BrowserPlugin {
+  let LOG: Logger;
+
+  return {
+    activateBrowserPlugin: (tracker: BrowserTracker) => {
+      _trackers[tracker.id] = tracker;
+    },
+
+    logger: (logger: Logger) => {
+      LOG = logger;
+    },
+
+    filter: (payload) => {
+      if (hasMobileInterface() === true) {
+        LOG.debug(`Payload (event ID: ${payload.eid}) was filtered out and forwarded to WebView tracker.`);
+
+        let atomicProperties = {
+          eventName: payload.e as string,
+          trackerVersion: payload.tv as string,
+          useragent: payload.ua as string,
+          url: payload.url as string,
+          title: payload.page as string,
+          referrer: payload.refr as string,
+          category: payload.se_ca as string,
```

Comment:
Some of these properties are optional so it's not completely correct to always cast to `string`. Rather it would be more correct to cast `as string | undefined`. It doesn't really change anything in practice but at least the types will be more representative in Typescript.